### PR TITLE
Build script for Flog

### PR DESCRIPTION
**Pull request by Sapana-Khemkar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: f/flog/flog_ubi_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+PACKAGE_NAME=flog
+PACKAGE_VERSION=${1:-v4.6.2}
+PACKAGE_URL=https://github.com/seattlerb/flog
+
+yum install -y git wget curl ruby ruby-devel rubygem-rake procps libcurl-devel libffi-devel sqlite sqlite-devel nodejs nodejs-devel nodejs-packaging 
+
+yum install -y --allowerasing gcc gcc-c++ yum-utils make automake autoconf libtool gdb* binutils rpm-build gettext 
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
```
Comment (seth-priya) [inline f/flog/flog_ubi_8.4.sh:30]: please remove unwanted repositories and packages

File: f/flog/flog_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Maintainer	: Sapana Khemkar {Sapana.khemkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
```
Comment (seth-priya) [inline f/flog/flog_ubi_8.4.sh:20]: Needs update as per new template

File: f/flog/flog_ubi_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: flog
+# Version	: 4.6.2
+# Source repo	: https://github.com/seattlerb/flog
+# Tested on	: UBI 8.4
+# Language      : Ruby
+# Travis-Check  : False
```
Comment (seth-priya) [inline f/flog/flog_ubi_8.4.sh:9]: this should be true?

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by Amit-Sirohi** [comment]
@seth-priya @kandarpamalipeddi can one of you please review this PR. Changes done as per review comments.

**Review by kandarpamalipeddi — APPROVED** [review]

**Review by seth-priya — APPROVED** [review]

**Pull request merged**